packages/openapi-v3-types/src/openapi-v3-spec-types.ts

```diff
@@ -21,6 +22,10 @@ export type OpenApiSpec = OAS3.OpenAPIObject;
 // tslint:disable-next-line:no-any
 export type ExtensionValue = any;
 
+export type SchemasObject = {
```

Comment:
I think this should be called `SchemaObject` (singular Schema)? The old code has `export interface SchemaObject`.

Reply:
There are two types: SchemaObject (for schema definitions) and SchemasObject (for `schemas` of the spec).